app_internal_test.go

```diff
@@ -115,3 +117,37 @@ func TestAnnotationError(t *testing.T) {
 	assert.ErrorIs(t, err, wantErr)
 	assert.Contains(t, err.Error(), wantErr.Error())
 }
+
+// TestDisableSignalHandler tests disabling the default signal handler.
+// It's an internal test to allow us replace the app.receivers.notify function.
+func TestDisableSignalHandler(t *testing.T) {
+	for _, disableSignalHandler := range []bool{false, true} {
```

Comment:
Use a map[string]bool

So you can set `testname` here in the `for` loop, then no if needed

Reply:
No preference. Happy to change if maintainers prefer.